qa/tasks/ses_qa.py

```diff
@@ -146,13 +146,20 @@ def drive_replace_initiate(self, **kwargs):
         In order to "hide" an existing disk from the ceph.c_v in teuthology
         the disk is formatted and mounted.
         """
-        # :TODO: make this random and find correct disk device
-        osd_id = "2"
+        from random import randint
+
         total_osds = self.master_remote.sh(number_of_osds_in_cluster)
-        assert int(total_osds) == 4, "Unexpected number of osds {} (expected 4)".format(total_osds)
+        osd_id = randint(0, int(total_osds) - 1)
```

Comment:
I'm still against of randomization here, since developers will be complaining of random nature of failures. And it will be complicated for automated systems to detect the failure is the same because failing message will be different in the logs each sequential run.